src/dotnet/commands/dotnet-migrate/TemporaryDotnetNewTemplateProject.cs

```diff
@@ -65,8 +65,15 @@ private ProjectRootElement GetMSBuildProject()
 
         private void RunCommand(string commandToExecute, IEnumerable<string> args, string workingDirectory)
         {
+            List<string> argsWithDirectory = new List<string>(args);
```

Comment:
Can we do this back up on line 53? It seems we have the necessary context there, and I'd like to avoid generating args in two places if we can.